app/grants/management/commands/subminer.py

```diff
@@ -65,19 +61,20 @@ def process_subscription(subscription, live):
                 if live:
                     txid = subscription.do_execute_subscription_via_web3()
                     print(f"   -- *waiting for mine* (txid {txid}) ")
-                    while not has_mined(txid, subscription):
+                    while not has_tx_mined(txid, subscription.grant.network):
                         time.sleep(10)
-                    status, __ = get_tx_status(txid, subscription.grant.network, timezone.now())
+                        print("   -- *waiting 10 seconds*")
+                    status, timestamp = get_tx_status(txid, subscription.grant.network, timezone.now())
```

Comment:
We're still not using `timestamp` anywhere here.

Can you revert this back to:  `status, __ = get_tx_status(txid, subscription.grant.network, timezone.now())`